analyzers/Crowdsec/crowdsec_analyzer.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from cortexutils.analyzer import Analyzer
+from crowdsec_api import Crowdsec
+from datetime import datetime
+
+
+class CrowdsecAnalyzer(Analyzer):
+    def __init__(self):
+        Analyzer.__init__(self)
+        self.crowdsec_key = self.get_param("config.api_key", None, "Missing Crowdsec API key")
+        self.crowdsec_client = None
+        self.verbose_taxonomies = self.get_param("config.verbose_taxonomies", False)
+        self.polling_interval = self.get_param("config.polling_interval", 60)
+
+    def summary(self, raw):
+        taxonomies = []
+        namespace = "Crowdsec"
+        levelinfo = "info"
+        levelorange = "suspicious"
+        levelgreen = "safe"
+
+        if 'as_name' in raw:
+                taxonomies.append(self.build_taxonomy(levelinfo, namespace, 'ASN', raw['as_name']))
+
+        if 'ip_range_score' in raw:
+                taxonomies.append(self.build_taxonomy(levelinfo, namespace, 'Score', raw['ip_range_score']))
+
+        if 'history' in raw:
+                taxonomies.append(self.build_taxonomy(levelinfo, namespace, 'LastSeen', raw['history']['last_seen']))
+
+        if 'attack_categories' in raw:
+                for attack in raw['attack_categories'] :
```

Comment:
did you mean attack_details ?
https://crowdsecurity.github.io/cti-api/#/default/get_smoke__ip_

Reply:
arf yes, endpoint has changed

so yes attack_details instead of attack_categories